api/api.ipr

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Today, it's fully generated from build.gradle. However, if you dork around with intellij settings manually, they will be reflected here.

Whenever we introduce new libraries these will get automatically updated, which is slightly annoying; but if we want to have other developers using IntelliJ share settings that aren't generated from build.gradle, it'd be nice to have these submitted. And it's also nice for me to be able to restore project state if my workspace gets blown away for some reason.